nerfstudio/models/splatfacto.py

```diff
@@ -212,6 +219,7 @@ def populate_modules(self):
         else:
             means = torch.nn.Parameter((torch.rand((self.config.num_random, 3)) - 0.5) * self.config.random_scale)
         self.xys_grad_norm = None
+        self.max_2Dsize = None
```

Comment:
nit: `max_2d_size`?

Reply:
we could also use a more descriptive word here than "size", from my skim I'm not 100% sure what units or what this is

Reply:
This part belongs to older version of splatfacto that culls screen size too bigs. I add it back in this new one as this makes the results look better.